code/_defines/items.dm

```diff
@@ -1,8 +1,15 @@
 #define ITEM_SLOT_RING	"ring"
 #define ITEM_SLOT_HANDS	"hands"
+#define ITEM_SLOT_HEAD	"head"
+#define ITEM_SLOT_CHEST	"chest"
+#define ITEM_SLOT_BELT	"belt"
+#define ITEM_SLOT_LEGS	"legs"
+#define ITEM_SLOT_CLOTHING	"clothing"
 
 #define RARITY_COMMON		0
 #define RARITY_UNCOMMON		1
 #define RARITY_RARE			2
 #define RARITY_EPIC			3
-#define RARITY_LEGENDARY	4
\ No newline at end of file
+#define RARITY_LEGENDARY	4
+#define RARITY_ARTIFACT	    5
+#define RARITY_CURSED       6
```

Comment:
I am not sure if cursed should be a rarity, also can you explain the idea with "artifact"?

Reply:
special effects should be handled in a different way, and legendary could as well be "artifact". Cursed goes in the same boat as special effects, there should be some system for adding modifiers or similar to items which handles curses, etc.